andela/travel_tool_front
# #161922350 Setup end to end test

#### What does this PR do?
- Set up end to end test


#### Description of Task to be completed?
- install library and configure package.json

#### How should this be manually tested?
- Make sure the server is running both frontend and backend
- open another terminal  
- run `yarn end2end` to run the test with browser mode
- run `yarn end2end:headless` to run test only in the console


#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- [161922350](https://www.pivotaltracker.com/story/show/161922350)

#### Screenshots (if appropriate)
- None

#### Questions:
- None
